jetty-server/src/main/java/org/eclipse/jetty/server/ForwardedRequestCustomizer.java

```diff
@@ -512,12 +512,7 @@ public void customize(Connector connector, HttpConfiguration config, Request req
         {
             port = requestURI.getPort();
         }
-        if (port == MutableHostPort.IMPLIED) // is implied
-        {
-            // get Implied port (from protocol / scheme) and HttpConfiguration
-            int defaultPort = 80;
-            port = proto.equalsIgnoreCase(config.getSecureScheme()) ? getSecurePort(config) : defaultPort;
-        }
+        // if (port == MutableHostPort.IMPLIED) // is implied, no port change needed
```

Comment:
can you make this comment not look like commented out code.  It is good info to have, but I found the format confusing

Reply:
True. will fix.